### ISSUE TITLE
Question Regarding Reproducibility of "Time-CMA" Paper Results

### ISSUE THREAD

**Issue by lbw-czj** [ChenxiLiu-HNU/TimeCMA#13]
Thank you for making your code and datasets publicly available; it is an invaluable resource for the community.

I have been diligently working on reproducing the experimental results presented in your paper using the code from your official GitHub repository. I have carefully followed the setup instructions in your documentation, using the provided datasets and configuring my environment accordingly.

My setup is almost identical to the one specified (PyTorch 2.1.2). The only minor difference is the CUDA version; I am using CUDA 11.8, whereas the documentation recommends CUDA 12.1. While I understand this difference is generally not expected to cause significant variations, I have unfortunately been unable to achieve the same results as reported in your paper.

I would be extremely grateful if you could provide some guidance on this matter. To help pinpoint the issue, I was wondering:

Was the random seed used for generating the final paper results different from the one currently available in the GitHub repository's code?
Are there any other crucial hyperparameter adjustments or configurations that were used for the paper's experiments which might differ from the default settings in the code?
Are there any other specific steps or details for reproduction that might not be explicitly mentioned in the documentation that you would recommend I look into?
Any insights you could share would be immensely helpful for my research. Thank you very much for your time and consideration, and again for your outstanding contribution to the field.

**Comment by 1Liyq** [comment]
I have the same problem. The results obtained through training are higher by 0.01 to 0.03 compared to the results presented in the paper.

**Comment by kr2122** [comment]
> I have the same problem. The results obtained through training are higher by 0.01 to 0.03 compared to the results presented in the paper.我也有同样的问题。通过训练获得的结果比论文中展示的结果高 0.01 到 0.03。

yes me too

**Comment by createfree6** [comment]
<img width="418" height="185" alt="Image" src="https://github.com/user-attachments/assets/73588293-6093-4258-8eac-5e3251852e2e" />
I have the same problem. ETTm1训练结束的结果，是不是源代码有什么地方没设置对。求助！！！！！！！！！！

**Comment by rrrrrrrrrice** [comment]
> <img alt="Image" width="418" height="185" src="https://private-user-images.githubusercontent.com/236814784/520362651-73588293-6093-4258-8eac-5e3251852e2e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU5NTI4MjgsIm5iZiI6MTc2NTk1MjUyOCwicGF0aCI6Ii8yMzY4MTQ3ODQvNTIwMzYyNjUxLTczNTg4MjkzLTYwOTMtNDI1OC04ZWFjLTVlMzI1MTg1MmUyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxN1QwNjIyMDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTViZDNiODEyN2JiM2NiODI4ZWU4NmQwN2ZmZGQwMmM4NWI2NDQwMDdhOGY4OWIwNzk2MjllYWUxMDVmMDViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sDrwZ9fPi_diiG3WpPdAmK7DM1lH54YOiUukEAVPDq4"> I have the same problem. ETTm1训练结束的结果，是不是源代码有什么地方没设置对。求助！！！！！！！！！！

我和你的复现结果差不多 我还做了消融实验 把llm去掉之后的结果和论文里的结果很接近

**Comment by rrrrrrrrrice** [comment]
> <img alt="Image" width="418" height="185" src="https://private-user-images.githubusercontent.com/236814784/520362651-73588293-6093-4258-8eac-5e3251852e2e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjU5NTI4MjgsIm5iZiI6MTc2NTk1MjUyOCwicGF0aCI6Ii8yMzY4MTQ3ODQvNTIwMzYyNjUxLTczNTg4MjkzLTYwOTMtNDI1OC04ZWFjLTVlMzI1MTg1MmUyZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMjE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTIxN1QwNjIyMDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTViZDNiODEyN2JiM2NiODI4ZWU4NmQwN2ZmZGQwMmM4NWI2NDQwMDdhOGY4OWIwNzk2MjllYWUxMDVmMDViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sDrwZ9fPi_diiG3WpPdAmK7DM1lH54YOiUukEAVPDq4"> I have the same problem. ETTm1训练结束的结果，是不是源代码有什么地方没设置对。求助！！！！！！！！！！
这是我复现的结果
<img width="1442" height="452" alt="Image" src="https://github.com/user-attachments/assets/1c25d7f4-285c-4ccf-95c9-c30c9860638d" />
